apps/twitch-bot/internal/service/database/postgresql/postgresql.go

```diff
@@ -19,7 +19,7 @@ type PostgreSQL struct {
 
 func NewPostgreSQL() database.Database {
 	dsn := os.Getenv("DATABASE_URL")
-	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{Logger: logger.Default.LogMode(logger.Info)})
+	db, err := gorm.Open(mysql.Open(dsn), &gorm.Config{Logger: logger.Default.LogMode(logger.Info)})
```

Comment:
🤔

Reply:
Let's create a ticket to make sure we are not mixing postgres and mysql.

Reply:
https://github.com/senchabot-dev/monorepo/issues/262